controller.py

```diff
@@ -168,6 +170,22 @@ def register(self, do_get=HTTP.get_with_timeout):
         SHELF_REL = "http://opds-spec.org/shelf"
 
         auth_response = None
+        def get_opds_links(response):
+            type = response.headers.get("Content-Type")
+            if type == "application/opds+json":
```

Comment:
OPDS 2 doesn't have different kinds of feeds like OPDS 1, so it's not necessary. I guess it wouldn't hurt to have it anyway.